### PR TITLE
Workshop feedback

### PR DESCRIPTION
**Pull request by romofel**
Pr addressing the feedback points of the workshop

  - [x] readme under "Required Configuration Options" does not list the property to supply the actual key.
  - [x] The way the yarn and nom commands are stacked without headers is confusing
  - [x] What is PATH_GENERATE_TOKEN=/value in the .env example? Is it tokenEndpoint referenced in props table?
  - [x] Same with SHIPENGINE_TENANT and "tenantId"?
  - [x] How do I actually make a request to this service? A sample curl command would be very helpful.
  - [x] Readme should reference https://www.shipengine.com/docs/partners/jwt/ not just Elements Guide
  - [x] Default port for token signing app should match default value for TOKEN_ENDPOINT in example SDK HTML snippet
  - [x] Add CORS (allow all) to token signing app
  - [x] Add a script to signing app to generate a config from prompted values

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,34 +17,74 @@ Example application for handling ShipEngine Elements authentication token reques
...
 npm dev
 ```
 
+Start with yarn in dev mode (watch mode)
+
 ```bash
 yarn dev
 ```
 
+3. Request a token from the application
```
Comment (luxehahn) [inline README.md:60]: A token can be generated without passing the tenant ID? and what would be the reason to exclude it?

File: README.md [hunk 2]
```diff
@@ -17,34 +17,74 @@ Example application for handling ShipEngine Elements authentication token reques
...
+
 ```bash
 npm dev
 ```
 
+Start with yarn in dev mode (watch mode)
+
 ```bash
 yarn dev
 ```
```
Comment (luxehahn) [inline README.md:58]: A little chunky here. These can be consolidated into either npm or yarn given that the scripts are the same and then could add a blurb stating to replace with npm/yarn for preference if you think that's necessary.

File: README.md [hunk 3]
```diff
@@ -96,11 +139,10 @@ Example json configuration file
   "privateKeyFileName": "private.pem",
   "platformTokenIssuer": "json issuer",
   "platformTokenKeyId": "json platform key id",
-  "shipenginePartnerApiKey": "json shipengine partner api key",
   "scope": "json scope",
   "partnerId": "json partner id",
-  "tenant": "json tenant",
-  "port": 4444,
+  "tenantId": "json tenant",
```
Comment (luxehahn) [inline README.md:144]:
```suggestion
  "tenantId": "json tenant id",
```

File: README.md [hunk 4]
```diff
@@ -121,13 +163,17 @@ environment variables -> json file -> config.js defaults
 | **platformTokenKeyId**  | Used as an identifier for the public RSA key that was submitted to your Account Manager. This value will be provided to you by your Account Manager after your public RSA key has been submitted.             |
 | **scope**               | Used to determine the permissions for the generated token. This value will be provided to you by your Account Manager.                                                                                        |
 | **partnerId**           | Your ShipEngine Account Identification, unique to your organization, not to be confused with a tenantId. This is provided for you by your Account Manager when you register to the ShipEngine Partner program |
+| **privateKey**          | Your generated RSA private key in string format                                                                                                                                                               |
+| **privateKeyFilename**  | File name of your generated RSA private key                                                                                                                                                                   |
```
Comment (luxehahn) [inline README.md:167]:
```suggestion
| **privateKeyFilename**  | The file name where your generated RSA private key is stored                                                                                                                                                                |
```

File: src/config.js [hunk 5]
```diff
@@ -1,8 +1,4 @@
 import convict from "convict";
-import path from "path";
-import * as url from "url";
-const __filename = url.fileURLToPath(import.meta.url);
-const __dirname = url.fileURLToPath(new URL(".", import.meta.url));
 
 export default convict({
   privateKeyFileName: {
```
Comment (luxehahn) [inline src/config.js:4]: should there be a comment here to remove the param if not used, or will the null value just be ignored? like if privateKeyFileName was not set in the .env

**Comment by romofel** [comment]
opting to simplify app #4

**Pull request closed without merging**